.cirrus.yml

```diff
@@ -71,3 +71,18 @@ fedora_selftests_task:
         matrix:
           - image: quay.io/avocado-framework/avocado-ci-fedora-32
           - image: quay.io/avocado-framework/avocado-ci-fedora-33
+
+#
+# This is a *partial* check for the behavior of the safeloader, that
+# compares it to Python's unittest behavior.  This is intended to help
+# in the resolution of https://github.com/avocado-framework/avocado/issues/4625
+# avoiding new regressions.  It should also be expanded (and/or converted
+# into a selftest) to include avocado-instrumented tests too.
```

Comment:
Any reason for not creating this inside selftests/*, so a make check could also catch this?

Reply:
No, I even mentioned that.  Doing that in v2.